### PR TITLE
Added unit test on middlewares settings

### PR DESCRIPTION
**Pull request by alshyra**

### CODE REVIEW

**Comment by alshyra** [comment]
@gautaz je suis prêt

File: src/broker-revolver.js [hunk 1]
```diff
@@ -6,57 +6,57 @@ const throttle = require('lodash.throttle');
 
 /**
  * @param {settingsUpdateEvent} settingsUpdateEvent
- * @param {brokerShell} brokerShell
+ * @param {() => import('moleculer').LoggerInstance} log
+ * @param {Function} nodeID
```
Comment (gautaz) [inline src/broker-revolver.js:10]: Function or function ?

**Review by gautaz — CHANGES_REQUESTED** [review]
A minor detail

File: src/broker-revolver.js (hunk 1 above)
Comment (alshyra) [reply src/broker-revolver.js:10]: Function :)

**Review by gautaz — APPROVED** [review]
LGTM

**Pull request merged**